.github/workflows/check-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Check Release
+on:
+  push:
+    branches:
+      - main
```

Comment:
The repo still has `master` as the default branch.

We could actually do the rename to `main`. Then update the other workflow here: https://github.com/QuantStack/jupyterlab-js-logs/blob/e081f732312db1f733c0d55672b037cffe217a80/.github/workflows/build.yml#L5